spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/source/TestSparkReaderDeletes.java

```diff
@@ -126,6 +126,7 @@ public static void startMetastoreAndSpark() {
         SparkSession.builder()
             .master("local[2]")
             .config("spark.appStateStore.asyncTracking.enable", false)
+            .config("spark.ui.liveUpdate.period", 0)
```

Comment:
This effectively disables async update.

Reply:
I think -1 is the value for disabling? 
https://spark.apache.org/docs/latest/configuration.html

Reply:
We don't want to disable update but async update. Please check the logic below.

https://github.com/apache/spark/blob/v3.5.0/sql/core/src/main/scala/org/apache/spark/sql/execution/ui/SQLAppStatusListener.scala#L457

Reply:
There's a chance that our metric is not updated yet when we check its existence if `spark.ui.liveUpdate.period > 0`

Reply:
I see, you are configuring it to zero because you want it to update always. Not just asynchronously (which can cause flakiness when we check within the update interval). 

Got it. Adding it in PR description can help for reviewers too.